packages/tests/src.ts/test-wallet.spec.ts

```diff
@@ -581,4 +589,99 @@ describe("Wallet createAccount", function () {
         assert.ok(receipt.accountAddress.match(new RegExp(/^0x/)), "accountAddress has the correct format");
     }).timeout(timeout);
+
+    it("Should transfer funds between accounts", async function() {
+        const acc1BalanceBefore = (await acc1Wallet.getBalance()).toNumber();
+        const acc2BalanceBefore = (await acc2Wallet.getBalance()).toNumber();
+        await acc1Wallet.sendTransaction({
+            to: acc2Wallet.account,
+            value: 1,
+        });
+        const acc1BalanceAfter = (await acc1Wallet.getBalance()).toNumber();
+        const acc2BalanceAfter = (await acc2Wallet.getBalance()).toNumber();
+
+        assert.strictEqual(acc1BalanceBefore > acc1BalanceAfter, true);
+        assert.strictEqual(acc2BalanceBefore < acc2BalanceAfter, true);
+        assert.strictEqual(acc2BalanceAfter - acc2BalanceBefore, 100000000);
+    }).timeout(timeout);
+
+    it("Should throw an error for crypto transfer with data field", async function() {
+        let exceptionThrown = false;
+        try {
+            await acc1Wallet.sendTransaction({
+                to: acc2Wallet.account,
+                value: 1,
+                data: '0x'
+            });
+        } catch (e: any) {
+            exceptionThrown = true;
+        }
+
+        assert.strictEqual(exceptionThrown, true);
+    }).timeout(timeout);
+
+    it("Should throw an error for crypto transfer with gasLimit field", async function() {
+        let exceptionThrown = false;
+        try {
+            await acc1Wallet.sendTransaction({
+                to: acc2Wallet.account,
+                value: 1,
+                gasLimit: 300000
+            });
+        } catch (e: any) {
+            exceptionThrown = true;
+        }
+
+        assert.strictEqual(exceptionThrown, true);
+    }).timeout(timeout);
+
+
+    it("Should throw an error for crypto transfer with missing to field", async function() {
+        let exceptionThrown = false;
+        try {
+            await acc1Wallet.sendTransaction({
+                value: 1
+            });
+        } catch (e: any) {
+            exceptionThrown = true;
+        }
+
+        assert.strictEqual(exceptionThrown, true);
+    }).timeout(timeout);
+
+    it("Should make a contract call with 'to' and 'value' with provided contract address as 'to'", async function() {
+        const abiGLDTokenWithConstructorArgs = JSON.parse(readFileSync('examples/assets/abi/GLDTokenWithConstructorArgs_abi.json').toString());
+        const contractByteCodeGLDTokenWithConstructorArgs = readFileSync('examples/assets/bytecode/GLDTokenWithConstructorArgs.bin').toString();
+        const contractFactory = new ethers.ContractFactory(abiGLDTokenWithConstructorArgs, contractByteCodeGLDTokenWithConstructorArgs, acc1Wallet);
+        const contract = await contractFactory.deploy(ethers.BigNumber.from('10000'), {gasLimit: 3000000});
+        await contract.deployed();
+
+        let exceptionThrown = false;
+        try {
```

Comment:
1. Why do we need to use `try` catch when we are not throwing errors. 
2. I  think that it would be better to assert the balance of the address before and after that or at least to assert that the tx went through successfully

Reply:
1. In case of errors, I want to get a test failure instead of a thrown exception
2. Yep, that's already done in [click here](https://github.com/hashgraph/hethers.js/pull/159/files?file-filters%5B%5D=.ts&show-viewed-files=true#diff-9e035e9ad5d460f4390a5b870b0ac7a667ad8efbeffda299b03ae54b9b3f01f3R687)

PS. Sorry about all the mess with the 2 branches, it would be better if all the work was done in one PR.